.github/workflows/php-decomposer.yml

```
@@ -201,3 +201,72 @@ jobs:
 
       - name: Run PHPStan
         run: phpstan analyze src --level ${{ inputs.phpstan-level }} --configuration tests/phpstan.neon.dist --autoload-file tests/phpstan.autoload.inc.php
+  cli_test:
+    runs-on: ubuntu-latest
+    name: "PHP-${{ matrix.php-versions }}: CLI Test"
+    needs: [ generate-matrix, phpunit ]
+    strategy:
+      matrix: ${{ fromJson(fromJson(needs.generate-matrix.outputs.matrix)) }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Cache decomposer dependencies
+        id: cache-decomposer
+        uses: actions/cache@v4
+        with:
+          path: |
+            ${{ github.workspace }}/vendor
+            ${{ github.workspace }}/decomposer.autoload.inc.php
+          key: ${{ runner.os }}-${{ hashFiles('decomposer.json') }}
+          restore-keys:
+            ${{ runner.os }}-
+
+      - name: Decomposer checkout
+        uses: actions/checkout@v4
+        with:
+            repository: 'move-backend/decomposer'
+
+            # Relative path under $GITHUB_WORKSPACE to place the repository
+            path: 'decomposer'
+
+      - if: ${{ steps.cache-decomposer.outputs.cache-hit != 'true' }}
+        name: Prepare decomposer target directory
+        run: mkdir -p "$GITHUB_WORKSPACE/vendor"
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-versions }}
+          extensions: 'autoload_psr-pprkut/autoload-psr@0.2.0,pecl_http,uuid'
```

Make make these variable too

---

Should I make it like:
```suggestion
          extensions: '{{ inputs.php-extensions }},pecl_http,uuid'
```
or rather:
```suggestion
          extensions: '{{ inputs.php-run-extensions }}'
```

---

I'd stay consistent with the `php-tests`, where we also use it for all extensions